### PR TITLE
Refactors janitor cartridge to a tablet app

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Helps with https://github.com/tgstation/tgstation/pull/65755

Removes old janitor cartridge app and replaces it with a tablet one.
Also adds the pimpin' ride to the list of tracked items, too.
Makes Janitors spawn with said app, too.

image

## Why It's Good For The Game

- Janitor cartridge app checks the WHOLE world for stuff, this is an improvement from that
- Also makes the tablet the janitor spawns with, actually useful (beyond downloading arcade console while I wait to be called somewhere).
- Allows Janitors to more easily track their janitor equipment, since using pure coordinates sucked. This is the last place where sole GPS coordinates were ever given, and it really shows.

## Changelog

:cl:
refactor: Janitor's custodial PDA cartridge has been replaced with a Custodial locator tablet app.
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/radar.dm [hunk 1]
```diff
@@ -248,6 +248,38 @@
 				return TRUE
 	return FALSE
 
+///////////////
+// Custodial //
+///////////////
```
Comment (SuperNovaa41) [inline code/modules/modular_computers/file_system/programs/radar.dm:253]: these are bad dont do these

File: code/game/objects/items/mop.dm [hunk 2]
```diff
@@ -23,7 +23,11 @@
 /obj/item/mop/Initialize(mapload)
 	. = ..()
 	create_reagents(max_reagent_volume)
+	GLOB.janitor_devices += src
 
+/obj/item/mop/Destroy(force)
+	. = ..()
+	GLOB.janitor_devices -= src
```
Comment (MrMelbert) [inline code/game/objects/items/mop.dm:30]:
```suggestion
/obj/item/mop/Destroy(force)
	GLOB.janitor_devices -= src
	return ..()
```

File: code/modules/modular_computers/file_system/programs/radar.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/modular_computers/file_system/programs/radar.dm:253]: i will sry

File: code/game/objects/items/mop.dm (hunk 2 above)
Comment (JohnFulpWillard) [reply code/game/objects/items/mop.dm:30]: man it was automatic

File: code/modules/modular_computers/file_system/programs/radar.dm [hunk 3]
```diff
@@ -248,6 +248,48 @@
...
+			continue
+		var/tool_name = custodial_tools.name
+
+		if(istype(custodial_tools, /obj/item/mop))
+			var/obj/item/mop/wet_mop = custodial_tools
+			tool_name = "[wet_mop.reagents.total_volume ? "Wet" : "Dry"] [wet_mop.name]"
+
+		if(istype(custodial_tools, /obj/structure/janitorialcart))
+			var/obj/structure/janitorialcart/janicart = custodial_tools
+			tool_name = "[janicart.name] - Water level: [janicart.reagents.total_volume] / 100"
```
Comment (MrMelbert) [inline code/modules/modular_computers/file_system/programs/radar.dm:281]:
```suggestion
			tool_name = "[janicart.name] - Water level: [janicart.reagents.total_volume] / [janicart.reagents.maximum_volume]"
```

Comment (MrMelbert) [reply code/modules/modular_computers/file_system/programs/radar.dm:281]: Also, can you even see this in the UI? In the screenshot it gets cut off so

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/file_system/programs/radar.dm (hunk 3 above)
Comment (JohnFulpWillard) [reply code/modules/modular_computers/file_system/programs/radar.dm:281]: I don't know how I would increase it, hovering over it shows the full text though.

**Pull request merged**